Orange/widgets/evaluate/owrocanalysis.py

```diff
@@ -561,6 +565,7 @@ def _setup_plot(self):
                 graphics = curve.avg_threshold()
                 self.plot.addItem(graphics.curve_item)
                 self.plot.addItem(graphics.confint_item)
+                graphics.curve_item.scene().sigMouseMoved.connect(self._on_mouse_moved)
```

Comment:
Connect the signal once in the `__init__` method: 
 `self.plotview.scene().sigMouseMoved.connect(self._on_mouse_moved)`